packages/azure-kusto-ingest/README.md

```diff
@@ -237,3 +236,30 @@ async function ingestFromFile() {
     await waitForStatus();
 }
 ```
+
+### Browser run
+
+The code is browser compatible with main differences in the run are that ingestFromFile accepts a Blob object and
+ingestFromStream accepts an ArrayBuffer. Fallback files are called '*.browser.ts'.
+
+To run a quick sample for azure-kusto-ingest we provided a webpack config which uses the file index.html which runs index.js as a script.
+First add to index.ts:
+
+import {main} from "../exampleBrowser"
+main().catch(e=>console.log(e)) 
+
+Then run: npm run webpack
```

Comment:
This only work from OUR npm file, since we don't publish the file it's useless for users.

We need at the end to publish a two .js files (for data and ingest), put them in a cdn or something and then the user can just include them (and also have an option for them to build manually).